go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	go.uber.org/zap v1.26.0
 	golang.org/x/net v0.19.0
 	google.golang.org/grpc v1.58.3
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/protobuf v1.33.0
```

Comment:
Is this upgrade to utilize new APIs? Maybe a helpful commit message may make it easier to for people to know happened here.

Reply:
Any chance we can split this up upgrades into seperate PRs? These changes to go.mod and go.sum make a lot of additional noise in the PR.